Lesson_04/l04_h01_if_else.py

```diff
@@ -0,0 +1,39 @@
+"""
+HW:3[Volkov Anton].
+
+we have four values w,x,y,z
+write if-elif-else statement that will search minimum value
+and print smth aka "'y' is minimum value'
+advice use python debugger and different values to check your algorithm
+w, x, y, z = 100, 200, 40, 300
+"""
+w, x, y, z = 100, 200, 40, 300
+# 1. I've written my solution use python debugger to check my algorithm
+if w <= x:
+    if w <= y:
+        if w <= z:
+            print("'w' is minimum value")
```

Comment:
The only one thing if duplicate of print messages that could be optimized by using 
TEMPLATE = '{} is minimum value'
print(TEMPLATE.format('w'))
print(TEMPLATE.format('x'))